docs/Configuration.md

```diff
@@ -31,6 +34,9 @@ The port to use when connecting to the blockchain network at runtime.  Default i
 **providerUrl**
 The url to use when connecting to the blockchain network at runtime.  Default is http://127.0.0.1.
 
+**truffleTimeout**
```

Comment:
why it called "truffle timeout" ?
I would like to see it as something like arc.js_tx_wait_timeout